request_test.go

```diff
@@ -347,7 +347,7 @@ func TestAllocateBodyProtocolVersions(t *testing.T) {
 				// TODO: JoinGroupRequest v6 is not supported, but expected for KafkaVersion 2.4.0
 				// apiKeyJoinGroup:                   6, // up from 6
 				// TODO: HeartbeatRequest v4 is not supported, but expected for KafkaVersion 2.4.0
-				// apiKeyHeartbeat:                   4, // up from 3
+				apiKeyHeartbeat: 4, // up from 3
```

Comment:
Minor but as you need to resolve conflicts on request_test.go anyway, you can remove the // TODO: comment for HeartbeatRequest v4?